frontend/public/locales/en/public.json

```diff
@@ -656,7 +656,7 @@
   "Help menu": "Help menu",
   "Select a workload": "Select a workload",
   "Add secret to workload": "Add secret to workload",
-  "addSecretDescription": "Add all values from <1></1>{{secretName}} to a workload as environment variables or a volume.",
+  "Add all values from <1></1>{{secretName}} to a workload as environment variables or a volume.": "Add all values from <1></1>{{secretName}} to a workload as environment variables or a volume.",
```

Comment:
Do we need a space between the icon and the secret name or is that handled by CSS?

Reply:
The resource icon has right margin for spacing. https://github.com/openshift/console/blob/master/frontend/public/components/_resource.scss#L9